### ISSUE TITLE
TurnLaneView does not handle some BannerComponents correctly

### ISSUE THREAD

**Issue by yunikkk** [mapbox/mapbox-navigation-android#3926]
**Android API:*any*
**Mapbox Navigation SDK version:*1.3.0*

[TurnLaneView](https://github.com/mapbox/mapbox-navigation-android/blob/release-v1.4/libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/turnlane/TurnLaneView.java#L60) seem to skip some  BannerComponents values, that leads to invalid lane indicator displayed.

Some examples that we've seen that are not handled : 
`slight right, right`, `left, right`

At the same time logic that parses lane indications [TurnLaneViewData.buildDrawData](https://github.com/mapbox/mapbox-navigation-android/blob/db085391c0a5e94d95e783e419c86b2cbf49d8f3/libnavigation-ui/src/main/java/com/mapbox/navigation/ui/instruction/turnlane/TurnLaneView.java#L71) does not support mentioned indication combinations. 

PS
We've noticed this by cloning TurnLane* classes to the project to modify them slightly and switching to Kotlin instead of java. Then nullable `TurnLaneViewData.drawMethod` was lost and we've got crashes for the mentioned BannerInstructions.

cc @LukasPaczos @JunDai @abhishek1508 @RingerJK since you've contributed to this classes

**Comment by LukasPaczos** [comment]
@abhishek1508 could you double-check whether this is resolved in v2?

**Comment by abhishek1508** [comment]
Yeah I came across this issue and I am already working on resolving it.
cc @LukasPaczos

**Comment by Guardiola31337** [comment]
Where are we here @abhishek1508?

**Comment by abhishek1508** [comment]
This is a beast in itself because there are many turn lane icons that needs to be addressed in both iOS/Android platforms. We are still trying to figure out all the possible combinations that might happen but it's gonna take time
cc @Guardiola31337
